CHANGELOG.md

```diff
@@ -1,3 +1,161 @@
+	[#954](https://github.com/weaveworks/scope/pull/954)
+	[#984](https://github.com/weaveworks/scope/pull/984)
+- Gather file descriptors as process metric
+	[#961](https://github.com/weaveworks/scope/pull/961)
+- Show Docker Labels in their own table in details panel
+	[#904](https://github.com/weaveworks/scope/pull/904)
+	[#965](https://github.com/weaveworks/scope/pull/965)
+- Improve highlighting of selected topology
+	[#936](https://github.com/weaveworks/scope/pull/936)
+  [#964](https://github.com/weaveworks/scope/pull/964)
+- Details: only show important metadata by default, expand the rest
+	[#946](https://github.com/weaveworks/scope/pull/946)
+- Reorder the children tables in the details panel by importance
+	[#941](https://github.com/weaveworks/scope/pull/941)
+- Shorten docker container and image IDs in the details panel.
+	[#930](https://github.com/weaveworks/scope/pull/930)
+- shorten some details panel labels which were truncated
+	[#940](https://github.com/weaveworks/scope/pull/940)
+- Feed sparkline data via buffer to keep them moving
+	[#795](https://github.com/weaveworks/scope/pull/795)
+- Add Container Count column to container images table
+	[#919](https://github.com/weaveworks/scope/pull/919)
+- Periodically check for newer versions of scope.
+	[#907](https://github.com/weaveworks/scope/pull/907)
+- Rename Applications -> Process, sort topologies by rank.
+	[#866](https://github.com/weaveworks/scope/pull/866)
+- Rename 'by hostname' to 'by dns name'
+	[#856](https://github.com/weaveworks/scope/pull/856)
+- Add container uptime and restart count to details panel.
+	[#853](https://github.com/weaveworks/scope/pull/853)
+- Use connection directions from conntrack (no UI support yet)
+	[#967](https://github.com/weaveworks/scope/pull/967)
+
+Performance enhancements:
+- Improve codec performance
+	[#916](https://github.com/weaveworks/scope/pull/916)
+  [#1002](https://github.com/weaveworks/scope/pull/1002)
+  [#1005](https://github.com/weaveworks/scope/pull/1005)
+  [#980](https://github.com/weaveworks/scope/pull/980)
+- Reduce amount of objects allocated by the codec
+	[#1000](https://github.com/weaveworks/scope/pull/1000)
+- Refactor app for multitenancy
+	[#997](https://github.com/weaveworks/scope/pull/997)
+- Improve performance of docker stats obtention
+	[#989](https://github.com/weaveworks/scope/pull/989)
+- Rate-limit reading proc files
+	[#912](https://github.com/weaveworks/scope/pull/912)
+- Add debug logging
+	[#935](https://github.com/weaveworks/scope/pull/935)
+- Compile k8s selectors once (not for each pod)
+	[#918](https://github.com/weaveworks/scope/pull/918)
+- Minimize race between proc's net/tcp{,6} and fd/*
+	[#905](https://github.com/weaveworks/scope/pull/905)
+- Fix reading of network namespace inodes
+	[#898](https://github.com/weaveworks/scope/pull/898)
+
+Bug fixes:
+- Use TCP for weave dns to fix autoclustering
+  [#1038](https://github.com/weaveworks/scope/pull/1038)
+- Add ping/pong to websocket protocol
```

Comment:
... to prevent websocket connections being dropped when traversing loadbalancers.